ossapi/endpoints.py

```diff
@@ -2,35 +2,35 @@
 class ENDPOINTS():
     class GET_BEATMAPS():
         EXTENSION = "get_beatmaps"
-        REQUIRED = []  # this is according to https://github.com/ppy/osu-api/wiki
-        POSSIBLE = ["since", "s", "b", "u", "type", "m", "a", "h", "limit"]
+        REQUIRED = [["since"], ["s"], ["b"], ["u"], ["h"]]
+        POSSIBLE = ["type", "m", "a", "limit", "mods"]
```

Comment:
you deleted too much here in the last commit

Reply:
did I? I can't find it

Reply:
```suggestion
        POSSIBLE = ["since", "s", "b", "u", "type", "m", "a", "h", "limit", "mods"]
```
you can see it above my comment? ![](https://i.imgur.com/xAks9hY.png)

Reply:
oh for some reason I thought parameters should only be in required or possible. I do think that would be a good change but it's not how we do it now so